README.md

```diff
@@ -243,9 +250,9 @@ Transactions can be created via the  `issue-genesis-utxo-expenditure` & `issue-u
 
 The easiest way to create a transaction is via the `scripts/issue-genesis-utxo-expenditure.sh` script as follows:
 
-`./scripts/issue-genesis-utxo-expenditure.sh transaction_file`
```

Comment:
Isn't `genesis_file` arg redundant here ? I think it will create one in `defaults/liveview` by default